locale/fr-FR/zotero-better-bibtex.dtd

```diff
@@ -33,7 +33,7 @@
 <!ENTITY better-bibtex.ErrorReport.reportID "Report ID :">
 <!ENTITY better-bibtex.ErrorReport.review "Les données ci-dessous sont sur le point d'être envoyées à Better BibTeX pour le débogage. Vous pouvez consulter les données pour voir s'il existe des informations sensibles que vous ne souhaitez pas envoyer.
 
-Si vous acceptez d'envoyer ces données, cliquez sur &quot;Suivant&quot; et publiez l'ID de débogage présenté dans l'écran suivant dans le suivi des problèmes à : 
+Si vous acceptez d'envoyer ces données, cliquez sur &quot;Suivant&quot; et publiez l'ID de débogage présenté dans l'écran suivant dans le suivi des problèmes à :
```

Comment:
I think the space was put there deliberately by the French translator. In any case, translations should not be changed in github but at crowdin